gcp-pipelines/x86_64-windows-msvc.yml

```diff
@@ -0,0 +1,16 @@
+steps:
+  - name: 'gcr.io/shaderc-build/shader-compiler-team:dxc-win-builder'
+    env:
+      - BUILD_ID=$BUILD_ID
+      - SOURCE_IMAGE=dxc-windows-vs22
+logsBucket: 'gs://public-github-building-logs'
+options:
+  logging: LEGACY
+  pool:
+    name: projects/shaderc-build/locations/europe-west1/workerPools/dxc
+artifacts:
+  objects:
+    location: 'gs://public-directx-shader-compiler/$COMMIT_SHA'
+    paths:
+      - bin/dxc.exe
```

Comment:
The AppVeyor artifacts grab a lot more than this. They’re basically full packages.

Reply:
CHanged the pipeline to take the artifact .zip files.
On linux, you have visibility on the content (content from install-distribution)
On Windows, it's done with this line:

```
xcopy C:\source\include\\dxc include\dxc /s /i /y || exit /b 1
 20 7z a dxc-artifacts.zip include Release\lib\d*.lib Release\bin\d*.exe Release\bin\d*.dll Release\bin\d*.pdb Release\bin\HLSLHost.exe C:\source\GIT-COMMIT.tx
    t || exit /b 1
```

For now the image files are not part of the public repo, but they could as nothing in them is private for now.